### ISSUE TITLE
Fix table output

### ISSUE THREAD

**Issue by chriscarrollsmith** [chriscarrollsmith/pyzotero-cli#18]
Table output is currently trash: $ zot items list -q "machine learning" --output table
+----------+-----------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| key      |   version | library                                                                                                                                                          | links                                                                                                                                                                                                         | meta                                                         | data                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+==========+===========+==================================================================================================================================================================+===============================================================================================================================================================================================================+==============================================================+======================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================+
| RJVQT3RR |      7092 | {'type': 'user', 'id': 17128156, 'name': 'chriscarrollsmith', 'links': {'alternate': {'href': 'https://www.zotero.org/chriscarrollsmith', 'type': 'text/html'}}} | {'self': {'href': 'https://api.zotero.org/users/17128156/items/RJVQT3RR', 'type': 'application/json'}, 'alternate': {'href': 'https://www.zotero.org/chriscarrollsmith/items/RJVQT3RR', 'type': 'text/html'}} | {'creatorSummary': 'Wade and Kevin Glynn', 'numChildren': 4} | {'key': 'RJVQT3RR', 'version': 7092, 'itemType': 'book', 'title': 'Hands-On Gradient Boosting with XGBoost and scikit-learn: Perform accessible machine learning and extreme gradient boosting with Python', 'creators': [{'creatorType': 'author', 'firstName': 'Corey', 'lastName': 'Wade and Kevin Glynn'}], 'abstractNote': '', 'series': '', 'seriesNumber': '', 'volume': '', 'numberOfVolumes': '', 'edition': '', 'place': '', 'publisher': '', 'date': '', 'numPages': '', 'language': '', 'ISBN': '', 'shortTitle': '', 'url': '', 'accessDate': '', 'archive': '', 'archiveLocation': '', 'libraryCatalog': '', 'callNumber': '', 'rights': '', 'extra': 'ASIN: B08GSSGTYF\nSource: Kindle Highlights Scraper', 'tags': [], 'collections': ['6KACA3VD'], 'relations': {}, 'dateAdded': '2025-05-18T14:47:30Z', 'dateModified': '2025-05-18T14:47:30Z'}    |
+----------+-----------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| 4N7V3ZMC |      7108 | {'type': 'user', 'id': 17128156, 'name': 'chriscarrollsmith', 'links': {'alternate': {'href': 'https://www.zotero.org/chriscarrollsmith', 'type': 'text/html'}}} | {'self': {'href': 'https://api.zotero.org/users/17128156/items/4N7V3ZMC', 'type': 'application/json'}, 'alternate': {'href': 'https://www.zotero.org/chriscarrollsmith/items/4N7V3ZMC', 'type': 'text/html'}} | {'creatorSummary': 'Theobald', 'numChildren': 3}             | {'key': '4N7V3ZMC', 'version': 7108, 'itemType': 'book', 'title': 'Machine Learning for Absolute Beginners: A Plain English Introduction (Third Edition) (Learn Machine Learning with Python Books for Beginners Book 1)', 'creators': [{'creatorType': 'author', 'firstName': 'Oliver', 'lastName': 'Theobald'}], 'abstractNote': '', 'series': '', 'seriesNumber': '', 'volume': '', 'numberOfVolumes': '', 'edition': '', 'place': '', 'publisher': '', 'date': '', 'numPages': '', 'language': '', 'ISBN': '', 'shortTitle': '', 'url': '', 'accessDate': '', 'archive': '', 'archiveLocation': '', 'libraryCatalog': '', 'callNumber': '', 'rights': '', 'extra': 'ASIN: B08RWBSKQB\nSource: Kindle Highlights Scraper', 'tags': [], 'collections': ['6KACA3VD'], 'relations': {}, 'dateAdded': '2025-05-18T14:47:42Z', 'dateModified': '2025-05-18T14:47:42Z'} |
+----------+-----------+------------------------------------------------------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
